storage/badger/model/storedChunkDataPack.go

```diff
@@ -0,0 +1,23 @@
+package badgermodel
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// StoredChunkDataPack is an in-storage representation of chunk data pack.
+// Its prime difference is instead of an actual collection, it keeps a collection ID hence relying on maintaining
+// the collection on a secondary storage.
+type StoredChunkDataPack struct {
+	ChunkID      flow.Identifier
+	StartState   flow.StateCommitment
+	Proof        flow.StorageProof
+	CollectionID *flow.Identifier
+}
+
+func (s StoredChunkDataPack) ID() flow.Identifier {
+	return s.ChunkID
```

Comment:
why is chunkid?

Reply:
This was the residue of another version of this implementation. Thanks for pointing out, I removed these methods in https://github.com/onflow/flow-go/pull/968/commits/f02ea8c8745cafcb526ccfded79c9fb2df3cab60. 

In an earlier draft of this PR, I was using `StoredChunkDataPack` out of the `badger` database as well. So, in some cases `StoredChunkDataPack` would be utilized as an alternative to the `ChunkDataPack`, and since `ChunkDataPack` is an entity, hence `StoredChunkDataPack` must be. And the `ID` and `CheckSum` methods of `StoredChunkDataPack` hence implemented the same as `ChunkDataPack` (otherwise we could not use them interchangeably: 

https://github.com/onflow/flow-go/blob/a1bee171b3a909c1aab808a559c244a6533ccd29/model/flow/chunk.go#L40-L51

However, at the final version of this PR, we decided to keep `StoredChunkDataPack` as an internal model for the `badger` database, and not to use it anywhere else, hence degrading it from an entity to a simple data structure.